src/ontology/mondo-ingest.Makefile

```diff
@@ -598,6 +598,19 @@ update-jinja-sparql-queries:
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_replace_annotation_based_mappings.py
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_mapping_annotations_violation.py
 
+
+#################
+##### Ad hoc ####
+#################
+tmp/ordo-subsets.tsv: $(COMPONENTSDIR)/ordo.owl
```

Comment:
## Delete `ordo-construct-subsets.ru`?
Would like Nico or Trish to confirm that it is OK to handle this issue / particularly as part of this PR. Would be a quick change, once a reviewer looks into the issue / confirms the plan.

Addresses:
- #514

Reply:
Better not as part of this PR - this is a different concern: the shape of the component, vs what we directly import into Mondo.